src/pudl/transform/eia923.py

```diff
@@ -846,7 +846,7 @@ def boiler_fuel(eia923_dfs, eia923_transformed_dfs):
         "sector_name",
         "fuel_unit",
         "total_fuel_consumption_quantity",
-        "respondent_frequency",
+        "reporting_frequency",
```

Comment:
What was your thinking with this new column name? Is it just that we aren't sure whether it corresponds to a respondent, or a plant, or some other entity? Or because it should be obvious from the entity table it ends up what kind of thing it is describing?

Reply:
ah this is the next task coming out of this PR that I haven't fully wrapped my head around. The EIA has both "Reporting Frequency" and "Respondent Frequency". These two columns have codes but slightly different definitions:

* "Designates the reporting frequency or frequencies for the plant. Alphanumeric."
* "Specifies what time period data has to be reported (i.e. monthly data or annual totals) and how often the power plant reports this data to EIA."

Reply:
Huh, I bet those are the same and they just changed the description. I don't think either of them really totally gets at it, but do the explanations in the coding table for the 3 possible codes fully explain it?